pages/education/apply-for-education-benefits/application/5495.md

```diff
@@ -1,17 +0,0 @@
----
-title: Dependents Request For Change
-heading: Dependents request for change
-entryname: 5495-edu-benefits
-layout: page-react.html
-description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
-order: 2
-hideFromSidebar: true
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/education/">Education</a></li>
-    <li><a aria-current="page" href="/education/how-to-apply/">Apply for education benefits</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-66529e1146e1311fd1c402fbd93b2977R5.